packages/contracts/contracts/da/BVM_EigenDataLayrFee.sol

```diff
@@ -22,6 +22,7 @@ contract BVM_EigenDataLayrFee is Initializable, OwnableUpgradeable, ReentrancyGu
     }
 
     function initialize(address _address) public initializer {
+        require(_address != address(0), "initialize: address is the zero address");
```

Comment:
`"initialize: address is the zero address"` -> `"initialize: can't set zero address as gasFeeAddress"`